app/javascript/components/MainContainer/Map/Map.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class Map extends Component {
+    render() {
+        return (<h1>Map placeholder</h1>);
+    }
+}
```

Comment:
Since this is a component with no usage of `componentDidMount`/`Update` or state, you can simplify it to:
```jsx
function Map() {
  return <h1>Map placeholder</h1>;
}

// or
const Map = () => <h1>Map placeholder</h1>;
```

In fact, for completely static components with no props at all, you could write:
```jsx
const Map = <h1>Map placeholder</h1>;
```
(But we normally don't do that, don't know why)